src/GitHub.Exports/Models/SimpleRepositoryModel.cs

```diff
@@ -18,6 +21,31 @@ public SimpleRepositoryModel(string name, UriString cloneUrl, string localPath =
             Icon = Octicon.repo;
         }
 
+        public SimpleRepositoryModel(string path)
+        {
+            if (path == null)
+                throw new ArgumentNullException("path");
```

Comment:
Wouldn't `NullGuard.Fody` catch this? If not, you could use the `nameof` operator here: `throw new ArgumentNullException(nameof(path));`

Yay C# 6!